app/controllers/register/establishers/individual/PostCodeLookupController.scala

```diff
@@ -49,7 +49,7 @@ class PostCodeLookupController @Inject()(
                                         )(implicit val ec: ExecutionContext) extends GenericPostcodeLookupController {
 
   private val title: Message = "messages__establisher_individual_address__title"
-  private val heading: Message = "messages__establisher_individual_address__title"
+  private val heading: Message = "messages__establisher_individual_address__heading"
```

Comment:
We should move this message into the view model constructor below (see review comment below)
```suggestion
```

Reply:
Both comments addressed.